doc/api/tls.md

```diff
@@ -201,7 +206,9 @@ called:
   error originated.
 
 ### Event: 'newSession'
-
+<!-- YAML
+added: v0.11.12
+-->
 The `'newSession'` event is emitted upon creation of a new TLS session. This may
```

Comment:
please keep space between added comment and text, this applies for other parts in this document